<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeleprinterSettings</class>
 <widget class="QDialog" name="TeleprinterSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fightclub Teleprinter Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settingsTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="bcastConfigTab">
      <attribute name="title">
       <string>Broadcast reciever</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QRadioButton" name="defaultSettingsRbttn">
         <property name="text">
          <string>Default settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="customSettingsRbttn">
         <property name="text">
          <string>Custo&amp;m settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="bcastPortLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="selBcastPort">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>45454</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bcastIDLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>ID:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="selBcastID">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>1234567</number>
           </property>
           <property name="value">
            <number>12345</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="appConfigTab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="customFontCBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Use custom font</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="customScaleCBox">
         <property name="text">
          <string>Scale font size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="chooseFontBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="chooseScaleBox">
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <number>80</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rclockConfigTab">
      <attribute name="title">
       <string>Roomclock</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="verticalSpacing">
        <number>3</number>
       </property>
       <item row="3" column="0">
        <widget class="QRadioButton" name="swissRClock">
         <property name="text">
          <string>Swiss</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="smoothRClock">
         <property name="text">
          <string>S&amp;mooth</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="sharpRClock">
         <property name="text">
          <string>Sharp</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="showSecHand">
         <property name="text">
          <string>Show second hand</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="sechandmvmtCaption">
         <property name="text">
          <string>Second hand movement:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <widget class="QPushButton" name="lockSettings">
       <property name="text">
        <string>Lock</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/breeze-icons/object-locked-16.png</normaloff>:/breeze-icons/object-locked-16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applySettings">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/breeze-icons/dialog-ok-apply-16.png</normaloff>:/breeze-icons/dialog-ok-apply-16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
